src/libraries/Common/src/Internal/Cryptography/UniversalCryptoDecryptor.cs

```diff
@@ -82,16 +82,21 @@ protected override unsafe int UncheckedTransformFinalBlock(ReadOnlySpan<byte> in
 
             if (_heldoverCipher == null)
             {
-                // Pinned so that we can avoid the `fixed` to stop the decrypt buffer from getting
-                // copied around by the GC.
-                ciphertext = GC.AllocateUninitializedArray<byte>(inputBuffer.Length, pinned: true);
+#if NET5_0
```

Comment:
Rather than defining this as an opt-in to new, define it as an opt-in to old (e.g. `#if NETSTANDARD`).  Otherwise when we bump 5 to 6 this will fall to the "legacy" path.

Reply:
Gotcha. Fixed, I think.